go/tools/testwrapper/wrap.go

```diff
@@ -0,0 +1,31 @@
+package testwrapper
+
+import (
+	"bytes"
+	"io"
+	"os"
+
+	"os/exec"
+)
+
+// WrapIfNecessary wraps the execution of the test
+func WrapIfNecessary() {
+	if os.Getenv("RAN_FROM_BAZEL") != "1" {
+		return
+	}
+	os.Unsetenv("RAN_FROM_BAZEL")
+
+	exitCode := 0
+	var stdoutBuf bytes.Buffer
+
+	args := append([]string{"-test.v"}, os.Args[1:]...)
```

Comment:
I just found in the CI results modifying this causes issues, I'll look into different ways of instructing the child process to run in verbose mode when using the wrapper.

Reply:
It looks like `//tests/legacy/transitive_data:go_default_test` is the only one affected, and that test is wrong. It would be broken with `go test -v`, too. The test should read `flag.Args()` instead of `os.Args`.

Reply:
#2348